src/CoreBundle/Factory/BackendTreeNodeFactory.php

```diff
@@ -0,0 +1,26 @@
+<?php
+namespace ChameleonSystem\CoreBundle\Factory;
```

Comment:
Please add the chameleon comment header and run csfixer.

Reply:
ok